Helper/ScopeSetting.php

```diff
@@ -20,6 +20,9 @@ class ScopeSetting extends \Magento\Framework\App\Helper\AbstractHelper
     const KLAVIYO_PASSWORD = 'klaviyo_reclaim_user/klaviyo_user/password';
     const KLAVIYO_EMAIL = 'klaviyo_reclaim_user/klaviyo_user/email';
 
+    const WEBHOOK_SECRET = 'klaviyo_reclaim_webhook/klaviyo_webhooks/webhook_secret';
+    const PRODUCT_DELETE_AFTER = 'klaviyo_reclaim_webhook/klaviyo_webhooks/using_product_update_webhook';
```

Comment:
the const is named DELETE and yet the string has the name 'using_product_update_webhook' Its my understanding that this webhook will only fire after a product is deleted, not updated

Reply:
good call, will rename 👍